src/style/Body.css

```diff
@@ -0,0 +1,5 @@
+.book__container {
+  margin: 0 6.25rem;
+
+  /* height: 10.625rem; */
```

Comment:
- [ ] It would be better if we remove commented out code :+1: It makes the codebase messy